src/containers/TagManager.jsx

```diff
@@ -0,0 +1,167 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Radium from 'radium';
+
+import Page from './Page';
+import ContentHeader from '../components/ContentHeader';
+import InPlaceEditor from '../components/forms/InPlaceEditor';
+import Tagger from '../components/forms/Tagger';
+
+import settings from '../settings';
+
+require('../../css/react-tag-input.css');
+
+@connect(state => {
+  return state.pipelines;
+})
+@Radium
+export default class TagManager extends React.Component {
+
+  componentWillMount() {
+    this.setState({ tags: [ 'top_commenter', 'banned', 'moderator', 'martian', 'ufo' ] });
+  }
+
+  confirmDeletion(index) {
+    var tagsCopy = this.state.tags;
```

Comment:
Objects and arrays are passed by reference in js. This wouldn't make a copy, just a pointer.